models/topics.models.js

```diff
@@ -5,4 +5,15 @@ exports.fetchTopics = () => {
     .then(({rows}) => {
         return rows;
     })
+}
+
+exports.fetchArticleById = (id) => {
+   return db.query('SELECT * From articles WHERE article_id = $1;', [id])
+   .then(({rows}) => {
+       if (rows.length === 0) {
+            return Promise.reject({status: 404, msg: "Article Not Found"});
+       }
+       return { article: rows[0] };
```

Comment:
Just seen this.

You can pretty much ignore my last comment but I'd advise sticking to creating the response object in either the controllers or the models instead of a mix of both :)